blog/2024/10-26-jellyfin-release-10.10.0/index.mdx

```diff
@@ -0,0 +1,70 @@
+
+## Breaking Changes
+
+* There are no known client support issues in the transition from 10.9.0 to 10.10.0 - all your clients should continue to work as-is without any issues or any forced upgrades, though this may change in the future.
+
+* We have deprecated Raspberry Pi V4L2 hardware transcoding support, and are looking to deprecate Raspberry Pi (i.e. ARMv7) in general for releases past 10.10.0. Raspberry Pi has always been a tricky platform for us, as it's popular but extremely poor in performance, and this has caused a lot of people a lot of headaches. With the RPi5 entirely removing a hardware acceleration engine and the aging of the RPi4, we are taking this action now to ensure it's widely known. This also means that 10.10.x will likely be the last major release of Jellyfin to support 32-bit ARM (`armhf` in Debuntu land), which we will likely drop for 10.11.0. If you are running Jellyfin on old ARM hardware, now is the time to start upgrading. See [PR #1148](https://github.com/jellyfin/jellyfin.org/pull/1148).
+
+* We now use the system temporary storage engine (e.g. `/tmp` on Linux) for storing temporary files, to allow us to leverage temporary ramdisks and the like and avoid cluttering potentially slow storage. This may cause issues if you specifically depended on the previous behaviour. See [PR #12226](https://github.com/jellyfin/jellyfin/pull/12226).
+
+* The server will now refuse to start if `ffmpeg` cannot be found, is an incorrect version, or does not function properly (missing extensions, etc.). With how critical `ffmpeg` is to Jellyfin, this has become very important to avoid mis-reported issues. This can be explicitly bypassed if needed. See [PR #12463](https://github.com/jellyfin/jellyfin/pull/12463).
+
+* Network paths in libraries have been fully removed and will no longer work. This functionality has been deprecated for a long time, and most of it was removed in 10.9.0, but this removes the remainder. See [PR #12446](https://github.com/jellyfin/jellyfin/pull/12446).
+
+## Cool New Features
+
+### Media Segments
+
+The Jellyfin server now supports the mangement of Media Segments. This means that we store some additional information for certain timespans on a video that clients can then use to provide additional actions. For example, when there is a Media Segment of the type intro, a client may display a button to skip that particular segment. For 10.10, we only provide the general structure to store those Media Segments, and you will still require a plugin to create them. We have created one plugin that does this, based on the Chapter names of a movie or Episode, which you can find in the Plugin catalog, and others are welcome. The Web interface fully supports skipping segments.
+
+### Trickplay
+
+We have significantly improved extraction by implementing keyframe extraction, which can boost image generating time significantly (around 100x, depending on configuration). This should mean no more multi-day Trickplay jobs!
```

Comment:
Might want to mention at the cost of timing accuracy of the frames?

Reply:
Did we also change the default (with a migration) so that it's generally "fast"? Or will it still be slow unless people explicitly change it.

Reply:
@gnattu this question is for you

Reply:
> Did we also change the default (with a migration) so that it's generally "fast"? Or will it still be slow unless people explicitly change it.

The default is not changed and the user needs to explicitly enable that due to compatibility reasons. I have a draft of documentation on this but I have not finished it yet